components/create_post/index.js

```diff
@@ -43,13 +43,15 @@ import {canUploadFiles} from 'utils/file_utils';
 import CreatePost from './create_post.jsx';
 
 function mapStateToProps() {
```

Comment:
Same as Saturn's comment on the CreateComment component here. Can we also rename this as well?